flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -66,36 +66,57 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
-      options.anrTimeoutIntervalMillis = (args["anrTimeoutIntervalMillis"] as Int).toLong()
-      // expose options for isAttachThreads?
-      options.isAttachStacktrace = args["attachStacktrace"] as Boolean
-
-      val enableAutoNativeBreadcrumbs = args["enableAutoNativeBreadcrumbs"] as Boolean
-      options.isEnableActivityLifecycleBreadcrumbs = enableAutoNativeBreadcrumbs
-      options.isEnableAppLifecycleBreadcrumbs = enableAutoNativeBreadcrumbs
-      options.isEnableSystemEventBreadcrumbs = enableAutoNativeBreadcrumbs
-      options.isEnableAppComponentBreadcrumbs = enableAutoNativeBreadcrumbs
-
-      options.maxBreadcrumbs = args["maxBreadcrumbs"] as Int
-      options.cacheDirSize = args["cacheDirSize"] as Int
-
-      val level = args["diagnosticLevel"] as String
-      val sentryLevel = SentryLevel.valueOf(level.toUpperCase(Locale.ROOT))
-      options.setDiagnosticLevel(sentryLevel)
-
-      val anrEnabled = args["anrEnabled"] as Boolean
+      (args["dsn"] as? String)?.let {
+        options.dsn = it
+      }
+      (args["debug"] as? Boolean)?.let {
+        options.isDebug = it
+      }
+      (args["environment"] as? String)?.let {
+        options.environment = it
+      }
+      (args["release"] as? String)?.let {
+        options.release = it
+      }
+      (args["dist"] as? String)?.let {
+        options.dist = it
+      }
+      (args["enableAutoSessionTracking"] as? Boolean)?.let {
+        options.isEnableSessionTracking = it
+      }
+      (args["autoSessionTrackingIntervalMillis"] as? Long)?.let {
+        options.sessionTrackingIntervalMillis = it
+      }
+      (args["anrTimeoutIntervalMillis"] as? Long)?.let {
+        options.anrTimeoutIntervalMillis = it
+      }
+      (args["attachThreads"] as? Boolean)?.let {
+        options.isAttachThreads = it
+      }
+      (args["attachStacktrace"] as? Boolean)?.let {
+        options.isAttachStacktrace = it
+      }
+      (args["enableAutoNativeBreadcrumbs"] as? Boolean)?.let {
+        options.isEnableActivityLifecycleBreadcrumbs = it
+        options.isEnableAppLifecycleBreadcrumbs = it
+        options.isEnableSystemEventBreadcrumbs = it
+        options.isEnableAppComponentBreadcrumbs = it
+      }
+      (args["maxBreadcrumbs"] as? Int)?.let {
+        options.maxBreadcrumbs = it
+      }
+      (args["cacheDirSize"] as? Int)?.let {
+        options.cacheDirSize = it
+      }
+      (args["diagnosticLevel"] as? String)?.takeUnless { options.isDebug == false }?.let {
+        val sentryLevel = SentryLevel.valueOf(it.toUpperCase(Locale.ROOT))
+        options.setDiagnosticLevel(sentryLevel)
+      }
+
+      val anrEnabled = (args["anrEnabled"] as? Boolean) ?: options.isAnrEnabled
       options.isAnrEnabled = anrEnabled
 
-      val nativeCrashHandling = args["enableNativeCrashHandling"] as Boolean
+      val nativeCrashHandling = (args["enableNativeCrashHandling"] as? Boolean) ?: false
```

Comment:
```suggestion
      val nativeCrashHandling = (args["enableNativeCrashHandling"] as? Boolean) ?: true
```

`enableNativeCrashHandling` default value is `true`, so I'd rather fallback to `true`